app/graphql/types/query_type.rb

```diff
@@ -38,6 +38,25 @@
     end)
   end
 
+  field :search, !Types::SearchResultType do
+    description 'Search Ontohub'
+
+    argument :query, !types.String do
+      description 'The query string'
+    end
+
+    argument :category, Types::SearchResult::CategoryType do
```

Comment:
Ah, okay. So filtering for the categories already happens on the server. This would definitely save some bandwidth and probably save some time during the search. It's good to have it restricted like this for the frontend, but API users might want to be able to query for multiple categories at once.

I suggest:
Allow to query for multiple categories at once in the API, but only specify one in a usual search query in the frontend. This also allows us to select multiple categories in the future if users give us such feedback.